### PR TITLE
update

### PR DESCRIPTION
**Pull request by SigireddyBalasai**
updated all user endpoints

### CODE REVIEW

**Comment by SigireddyBalasai** [comment]
trying

**Comment by SigireddyBalasai** [comment]
ok

**Comment by SigireddyBalasai** [comment]
ok

**Comment by SigireddyBalasai** [comment]
ok

**Comment by SigireddyBalasai** [comment]
ok

**Comment by SigireddyBalasai** [comment]
done?

**Comment by SigireddyBalasai** [comment]
k

**Comment by SigireddyBalasai** [comment]
a

**Comment by SigireddyBalasai** [comment]
ok

**Comment by SigireddyBalasai** [comment]
j

**Comment by SigireddyBalasai** [comment]
merge

File: infamous_imps/server/dba.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    """ Adds new message to the server"""
+    con = sqlite3.connect("data.db") 
+    cur = con.cursor()
+    cur.execute("insert into chat values (?,?,?)", (user,timestamp,text))
+    con.commit()
+    con.close()
+        
+        
+def add_user(name,hashed):
+    "Adds new player to the database"""
```
Comment (nikhiljohn10) [inline infamous_imps/server/dba.py]: Give 2 more double quotes at the start

File: infamous_imps/server/dba.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    """See all message from the server"""
+    con = sqlite3.connect("data.db") 
+    cur = con.cursor()
+    return cur.execute('SELECT * FROM server ORDER BY timestamp')
+def check_user(name,hashed):
+    """Check if given credentials are correct"""
+    con = sqlite3.connect("data.db") 
+    cur = con.cursor()
+    name = (name,)
+    c = cur.execute(f'SELECT * FROM auth where user = ?',name)
```
Comment (nikhiljohn10) [inline infamous_imps/server/dba.py]: This is the wrong syntax for f-string
Example: `print(f"{val}for{val} is a portal for {val}.")`

Rather remove 'f' from it if you do not want interpolation.

**Comment by SigireddyBalasai** [comment]
ok

**Comment by SigireddyBalasai** [comment]
retry

**Pull request closed without merging**